rebar.config.script

```diff
@@ -9,10 +9,20 @@ ErlOpts = if not HaveEQC -> []; true ->
     [{d, 'HAVE_EQC'}]
 end,
 
-case lists:keyfind(erl_opts, 1, CONFIG) of
+Config1 = case lists:keyfind(erl_opts, 1, CONFIG) of
     {erl_opts, Opts} ->
         NewOpts = {erl_opts, Opts ++ ErlOpts},
         lists:keyreplace(erl_opts, 1, CONFIG, NewOpts);
     false ->
         CONFIG ++ [{erl_opts, ErlOpts}]
+end,
+
+case os:cmd("echo 'int main(int argc, char *argv[]) {return 0;}' | cc -c -x c -o /dev/null -flto -") of
```

Comment:
We should respect the setting of `$CC` environment variable here (rather than use hardcoded `cc`) -- as does rebar port compiler.

Reply:
I pushed a change to grab CC from the environment with a default of "cc".

Reply:
Also noting that this changed to PR #186 is where those updates happened. Gonna close this one out.

Reply:
Because of the .travis.yml changes I had to make if that wasn't clear.